dev/com.ibm.websphere.appserver.features/visibility/public/websocket-1.0/com.ibm.websphere.appserver.websocket-1.0.feature

```diff
@@ -10,10 +10,9 @@ IBM-API-Package: javax.websocket; type="spec", \
  com.ibm.websphere.wsoc; type="ibm-api"
 IBM-ShortName: websocket-1.0
 Subsystem-Name: Java WebSocket 1.0
--features=com.ibm.websphere.appserver.servlet-3.1; ibm.tolerates:="4.0", \
-  com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0,8.0"
```

Comment:
Why remove the `eecompatible` feature here?

Reply:
Same as above